lib/container.ex

```diff
@@ -70,6 +70,29 @@ defmodule RemoteDockers.Container do
   end
   def list_all!(_), do: raise ArgumentError.exception("Invalid host config type")
 
+  @doc """
+  Build a new Container object.
+
+  ## Example:
+    ```elixir
+    iex> Container.new(DockerHostConfig.new("localhost", 2345), "my_container_id")
+    %Container{
+      docker_host_config: %DockerHostConfig{
+        hostname: "localhost",
+        port: 2345
+      },
+      id: "my_container_id"
+    }
+    ```
+  """
+  @spec new(DockerHostConfig.t, bitstring) :: RemoteDockers.Container
+  def new(%DockerHostConfig{} = docker_host_config, container_id) do
```

Comment:
I don't think you need that, Container will be construct for you (after a get list or a simple get)